README.md

```diff
@@ -391,6 +391,8 @@ Render all underlying links as follows:
 - Internal links use client-side navigation with smooth scrolling via [react-router-hash-link],
   and pass through the `v` URL parameter for UI version switching
 - External links open in a new tab
+- Attachment links open attachments in a new tab.
```

Comment:
Do we want this to open in a new tab? If not, we can unset the `target` in this scenario.

Reply:
I chose to open in new tab because when opening pdf links, you end up completely leaving the site

Reply:
Is this because you aren't passing through the `download` prop to `a`?

Reply:
Ah, failed

Reply:
I've amended this to open in the same tab